fe/fe-core/src/main/java/com/starrocks/catalog/Table.java

```diff
@@ -194,11 +194,14 @@ public static TableType deserialize(String serializedName) {
     // foreign key constraint for mv rewrite
     protected List<ForeignKeyConstraint> foreignKeyConstraints;
 
+    protected Map<PartitionKey, Long> partitionKeyToId;
```

Comment:
is olap table still need this?

Reply:
No need.  Only table implements `HiveMetaStoreTable`.

However this `HiveMetaStoreTable` is interface, and there are many implementations . 

So I think it's better to put it here.